src/OpenSearch/Client.php

```diff
@@ -114,7 +114,7 @@
  */
 class Client
 {
-    public const VERSION = '2.4.5';
+    public const VERSION = '2.4.4';
```

Comment:
@shyim this doesn't look right.

Reply:
In `util/template/client-class` and the generated `src/OpenSearch/Client.php`

Reply:
Maybe we can always set this back to `2.x` or similar after each release, then switch to a proper version when we do a release.

Reply:
Looks like we missed that step.